src/cli.js

```diff
@@ -220,6 +222,11 @@ function cli(inputArgs) {
         }
         opts.options = downstreamArgs.concat(unparsedArgs);
 
+        if (cmd == 'run' && args.list && cordova.raw.targets) {
+            cordova.raw.targets(opts).done();
```

Comment:
Code further down uses .call(null, ...), I'm not quite sure why, but unless there's a good reason not to follow that convention, we should probably follow it.

-- Please ask someone.